webapplication/aoi/management/commands/_ComponentExecutionHelper.py

```diff
@@ -34,9 +32,15 @@ def get_environment(request:Request) -> Dict[str, str]:
             'SENTINEL2_CACHE':os.path.join(
                 settings.NOTEBOOK_POD_DATA_VOLUME_MOUNT_PATH, 
                 str(settings.SATELLITE_IMAGES_FOLDER). \
-                    replace(os.path.commonpath([settings.SATELLITE_IMAGES_FOLDER, settings.PERSISTENT_STORAGE_PATH])+'/', '')
+                    replace(os.path.commonpath([settings.SATELLITE_IMAGES_FOLDER, settings.PERSISTENT_STORAGE_PATH])+'/', ''), 
+                f"request_{request.pk}_cache"
```

Comment:
Let's add a settings for this with a backward compatible default value

Reply:
fixed